core/src/test/scala/unit/kafka/server/FetchSessionTest.scala

```diff
@@ -201,7 +201,7 @@ class FetchSessionTest {
       context6.updateAndGenerateResponseData(respData2).error())
 
     // Close the incremental fetch session.
-    var prevSessionId = resp5.sessionId()
+    var prevSessionId = resp5.sessionId() //FIXME what should happen here?
```

Comment:
@cmccabe what was intended here, should it be a `val` or was the intent to update `prevSessionId` in the loop?

Reply:
It can be a `val` here.

Reply:
Just to clarify, the purpose of the loop is to verify that if we destroy and later re-create a session, we eventually get a new ID.  Although the chances of an ID collision are extremely low, I didn't want to write a nondeterministic unit test.

Reply:
Thanks for the explanation.